src/foam/u2/tag/TernarySwitch.js

```diff
@@ -0,0 +1,237 @@
+foam.CLASS({
```

Comment:
license header.

Reply:
I always forget that... I gotta make a git plugin or something that checks my new files